mangaki/mangaki/admin.py

```diff
@@ -76,6 +191,10 @@ class WorkAdmin(admin.ModelAdmin):
         'controversy',
     )
 
+    def get_queryset(self, request):
+        qs = super().get_queryset(request)
+        return qs.filter(redirect__isnull=True)  # Exclude phantom works
```

Comment:
Doesn't it make sense for the default `Work` queryset to exclude those so-called phantom works, with explicit overrides (via a `Work.all_objects` or some such) when we really want to see them appear -- which would probably be only when viewing a `Work` and redirecting to its canonical version?